[Digital Mathematics Curriculum](http://wikieducator.org/Digital_Math)

<a data-flickr-embed="true"  href="https://www.flickr.com/photos/kirbyurner/43027738942/in/album-72157695209731812/" title="P1050696"><img src="https://farm2.staticflickr.com/1764/43027738942_9f1de39507_z.jpg" width="640" height="427" alt="P1050696"></a><script async src="//embedr.flickr.com/assets/client-code.js" charset="utf-8"></script>

# Synergetics

"Synergetics" is the name Fuller eventually gave to his life-long philosophical explorations, which were distilled into a two-volume magnum opus, *Synergetics* and *Synergetics 2*, published by Macmillan in 1975 and 1979 respectively, with softcover editions, and a posthumous web-based edition to follow.  The process of distilling and publishing the two volumes was undertaken in collaboration with [E.J. Applewhite](http://www.grunch.net/synergetics/applewhites.html), who wrote *Cosmic Fishing* to chronicle their adventure.  Dr. Arthur Loeb, crystallographer and Renaissance man in his own right, wrote the Preface.

<a data-flickr-embed="true" href="https://www.flickr.com/photos/kirbyurner/3005064170/in/album-72157612330643992/" title="Fuller Syllabus"><img src="https://live.staticflickr.com/3252/3005064170_8fd4ff6631.jpg" width="500" height="375" alt="Fuller Syllabus"></a><script async src="//embedr.flickr.com/assets/client-code.js" charset="utf-8"></script>

At the core of Synergetics is Fuller's critique of ancient Greek metaphysics and its right-angled orthodoxies. He emphasizes the topological and structural advantages of the tetrahedron (simpler, stronger) over the cube, and devises a system of geometric relationships in the form of nested polyhedra dubbed "the concentric hierarchy" with the tetrahedron as his unit of volume.  His vocabulary is deliberately remote (250.30) as he seeks to relate tension, syntropy and gravity to their complements: compression, entropy and radiation.

Fuller regarded his "explorations in the geometry of thinking" (the book's subtitle) as the principle source of his inventions and a direct result of his commitment, undertaken in 1927, to serve humanity as a whole, a goal which necessitated a deep rethinking of our shared predicament and prospects aboard Spaceship Earth.  In retrospect, we see Fuller was in tune with the zeitgeist in terms of the topics he focused on:  sphere packing, vectorial energy models, topology, tensegrity, geodesic spheres and domes.  

Categorization of the work, in terms of genre, has proved elusive. Dr. Peter Suber of Earlham College eventually accepted it as a philosophy, based on Fuller's own conception of what it was: 

> The integration of geometry and philosophy in a single conceptual system providing a common language and accounting for both the physical and metaphysical. (251.50)

### Polyhedrons = Networks = Graphs

The ephemeralization of polyhedrons is noted by Peter Cromwell in his book *Polyhedra*.  From solids, they became wireframes, and wireframes, of nodes and edges, define graphs, likewise networks.

<a data-flickr-embed="true" href="https://www.flickr.com/photos/kirbyurner/3175452271/in/album-72157612330643992/" title="Reference Books"><img src="https://live.staticflickr.com/3340/3175452271_2b26da864f_w.jpg" width="300" height="400" alt="Reference Books"></a><script async src="//embedr.flickr.com/assets/client-code.js" charset="utf-8"></script>
<br />
<a data-flickr-embed="true" href="https://www.flickr.com/photos/kirbyurner/6335726352/in/album-72157612330643992/" title="S Module"><img src="https://live.staticflickr.com/6114/6335726352_902009df40.jpg" width="500" height="441" alt="S Module"></a><script async src="//embedr.flickr.com/assets/client-code.js" charset="utf-8"></script>

### Sphere Packing

Sphere packing is a sub form of uniform point distribution ala the ideal gas model (Avogadro et al), and in Synergetics this is especially easy starting from point masses at the sphere centers.  The [OEIS sequence](https://oeis.org/A005901) 1, 12, 42, 92... and it's [cumulative version](https://oeis.org/A005901) jump out as easy topics in programming.

*Linus Pauling archive, Oregon State University*:

<a data-flickr-embed="true" href="https://www.flickr.com/photos/kirbyurner/3049399298/in/album-72157612330643992/" title="Cuboctahedral Packing"><img src="https://live.staticflickr.com/3213/3049399298_3029466d95_w.jpg" width="400" height="300" alt="Cuboctahedral Packing"></a><script async src="//embedr.flickr.com/assets/client-code.js" charset="utf-8"></script>

*The Book of Numbers* by Conway and Guy, and *Gnomon* by Midhat Gazale suggest themselves at this point.  The goal is to keep going back and forth between lexical and graphical treatments of the same sequences, whether planar ("2D") or spatial ("3D").  

In the namespace of Synergetics, it's also OK to say "4D" to mean anything "prefrequency" (eternal, independent of scale) but then "time is the only dimension" in another sense (energetic).  

> Time is size and size is time. Time is the only dimension. In synergetics time-size is expressible as frequency. (960.03)

"Dimension" is a slippery concept to begin with, and relates to "power".

The sphere packing that coincides with what Fuller dubs the "isotropic vector matrix" is the well-known and much studied CCP (cubic close packing), nowadays proved a closest packing.

### Concentric Hierarchy

<a data-flickr-embed="true" href="https://www.flickr.com/photos/kirbyurner/51493803050/in/photolist-2msgpP9-2mskcCw-XvSatr-9dBJbo-9dqru5-8rBHhG-8ryEix-8rBHYW-7DSum6-7jw8zp-6XSpXY-2ypHhr" title="concentric_hierarchy"><img src="https://live.staticflickr.com/65535/51493803050_c1923b3ff5.jpg" width="500" height="375" alt="concentric_hierarchy"></a><script async src="//embedr.flickr.com/assets/client-code.js" charset="utf-8"></script>

Four equi-radius spheres pack inter-tangently to form a tetrahedron.  More pack to form other shapes, or the same shapes with higher frequency.  

We're back to the lexical and graphical rules of Sphere Packing, however now we're building a schematic around the concept of volume in particular, with the tetrahedron as unit volume.

In [1]:
"""
The concentric hierarchy can be described in terms of phi-scaled S & E modules: 

Tetrahedron :   1S6 +  1S3 
Cube        :   3S6 +  3S3 
Octahedron  :   4S6 +  4S3 
Rh Triac    :   5S6 +  5S3 
Rh Dodec    :   6S6 +  6S3
Icoashedron : 120E3 +  20E
Dodecahedron:   84E +  12E
VE          : 420S  + 100s3
SuperRT     : 480E  + 120e3
--------

From David Koski's paper:  Revisiting R.B. Fuller's S & E Modules
On-line:  http://coffeeshopsnet.blogspot.com/2017/06/koski-paper.html
"""
from math import sqrt as rt2

φ    = (rt2(5)+1)/2      # golden ratio
Syn3 = rt2(9/8)          # not to be confused with Smod

S     = (φ **-5)/2       # home base Smod
s3    = S  * φ**-3       # small s, phi down
S3    = S  * φ**3        # capital s, phi up
S6    = S3 * φ**3        # phi up yet again

E3    = (rt2(2)/8)       # Emod phi up
E     = E3 * (φ**-3)     # home base Emod
e3    = E * φ**-3        # Emod phi down

T     = 1/24

Tetrahedron       =   1*S6 +   1*S3 
Cube              =   3*S6 +   3*S3 
Octahedron        =   4*S6 +   4*S3 
Rh_Triac_T        =   5*S6 +   5*S3 # (volume = 5) 
Rh_Triac_E        = 120*E
Pent_Dodecahedron =  84*E3 +  12*E
Rh_Dodecahedron   =   6*S6 +   6*S3
Icosahedron       = 100*E3 +  20*E
Cuboctahedron     = 420*S  + 100*s3
SuperRT           = 480*E  + 120*e3

print(f""" 
Shape                Tetravolume
--------------------------------------------------------
SuperRT           : {SuperRT:10.6f}
Cuboctahedron     : {Cuboctahedron:10.6f}
Icosahedron       : {Icosahedron:10.6f} (Platonic)
Pent_Dodecahedron : {Pent_Dodecahedron:10.6f} (Platonic)
Rh_Dodecahedron   : {Rh_Dodecahedron:10.6f}
Rh_Triac_E        : {Rh_Triac_E:10.6f} (120 Es)
Rh_Triac_T        : {Rh_Triac_T:10.6f} (120 Ts)
Octahedron        : {Octahedron:10.6f} (Platonic)
Cube              : {Cube:10.6f} (Platonic)
Tetrahedron       : {Tetrahedron:10.6f} (Platonic)""")

print(f"""\
--------------------------------------------------------
E3                : {E3:10.6f} (phi up) 
E                 : { E:10.6f} (base)
T                 : { T:10.6f} 
e3                : {e3:10.6f} (phi down) 

S3                : {S3:10.6f} (phi up) 
S                 : { S:10.6f} (base) 
s3                : {s3:10.6f} (phi down)
--------------------------------------------------------""")

S_factor = S/E
SmallVE    = 20 * 1/8  # half D edges (=R)
SkewIcosa  = SmallVE * S_factor * S_factor
S_check    = (Octahedron - SkewIcosa)/24
print(f"S                 : {S:10.6f}")
print(f"S check           : {S_check:10.6f}")

 
Shape                Tetravolume
--------------------------------------------------------
SuperRT           :  21.213203
Cuboctahedron     :  20.000000
Icosahedron       :  18.512296 (Platonic)
Pent_Dodecahedron :  15.350018 (Platonic)
Rh_Dodecahedron   :   6.000000
Rh_Triac_E        :   5.007758 (120 Es)
Rh_Triac_T        :   5.000000 (120 Ts)
Octahedron        :   4.000000 (Platonic)
Cube              :   3.000000 (Platonic)
Tetrahedron       :   1.000000 (Platonic)
--------------------------------------------------------
E3                :   0.176777 (phi up) 
E                 :   0.041731 (base)
T                 :   0.041667 
e3                :   0.009851 (phi down) 

S3                :   0.190983 (phi up) 
S                 :   0.045085 (base) 
s3                :   0.010643 (phi down)
--------------------------------------------------------
S                 :   0.045085
S check           :   0.045085


From *Synergetics*:  Rhombic Triacontahedron (RT) made of 120 T or E modules (indistinguishable at this scale):

<a data-flickr-embed="true" href="https://www.flickr.com/photos/kirbyurner/4148457444/in/photolist-7jzVSm-97TTvV-7aEsJY-5zY9iA" title="Rhombic Triacontahedron"><img src="https://live.staticflickr.com/2662/4148457444_60e88eee55_w.jpg" width="400" height="312" alt="Rhombic Triacontahedron"></a><script async src="//embedr.flickr.com/assets/client-code.js" charset="utf-8"></script>

### Jitterbug Transformation

A twisting-contracting motion dynamically connects the concentric hierarchy polyhedrons, as various stations along the way. The cuboctahedron becomes the icosahedron, which in turn collapses to the octahedron, which may further simplify to the tetrahedron. 

Exactly how all this gets shown depends on the model used.

Once scrunched down to a tetrahedron, the theme becomes one of inside-outing through a zero, and re-expanding inside "Negative Universe".  Exactly what "Negative Universe" might be good for remains open to speculation and/or application.  The oscillation between positive and negative convergence / divergence suggests a "bow tie" (⋈) or "dorje" pattern (1033.664), intuited by Fuller as early as 1927.

<a data-flickr-embed="true" href="https://www.flickr.com/photos/kirbyurner/8393394058/in/photolist-2cxE3aG-2cxE31d-2dz7FmY-dMGkCE-5y2HNo" title="Jitterbug Transformation"><img src="https://live.staticflickr.com/8195/8393394058_b096cc1e20.jpg" width="500" height="375" alt="Jitterbug Transformation"></a><script async src="//embedr.flickr.com/assets/client-code.js" charset="utf-8"></script>
<br />
<a data-flickr-embed="true" href="https://www.flickr.com/photos/kirbyurner/8392298401/in/dateposted-public/" title="Snazzy"><img src="https://live.staticflickr.com/8096/8392298401_72299a4864_w.jpg" width="400" height="300" alt="Snazzy"></a><script async src="//embedr.flickr.com/assets/client-code.js" charset="utf-8"></script>

### Spinning Polyhedrons

Once the concentric hierarchy is established, around the Platonics and their dual combinations (e.g. octahedron + cube = rhombic dodecahedron, the cuboctahedron's dual), it's time to spin them around the obvious axes involving:  

* opposite vertices
* opposite face centers
* opposite edge mid-points  

The resulting great circles create characteristic and partially overlapping webs associated with each of the polyhedra.

For example, the cuboctahedron with 12 nodes, 24 edges and 14 faces, is the source of 6 + 12 + 7 = 25 great circles.  The icosahedron, with 12 nodes, 30 edges, and 20 faces, defines 6 + 15 + 10 = 31 great circles. But then the Jitterbug may contract to an icosahedron in two chirally symmetrical ways, and these two networks may be superimposed, along with that of the cuboctahedron (1042.05). 

The spherical geometry of higher frequency networks provides the dimensional characteristics of spheres and domes.

<a data-flickr-embed="true" href="https://www.flickr.com/photos/kirbyurner/51745621666/in/dateposted-public/" title="high_frequency_icosa"><img src="https://live.staticflickr.com/65535/51745621666_dbdc12da48_m.jpg" width="229" height="220" alt="high_frequency_icosa"></a><script async src="//embedr.flickr.com/assets/client-code.js" charset="utf-8"></script>


<pre>
V = 10 ff + 2 (f = frequency)
V + F = E + 2 (Euler's Law)
N = V - 2     (defining non-polar)
N:F:E = 1:2:3 (omnitriangulated)
</pre>

<a data-flickr-embed="true" href="https://www.flickr.com/photos/kirbyurner/479693617/in/photolist-dNwXux-dNCkJo-8awBAd-6eHz8h-5mMmQs-5dtZZf-B2cFHb-uewShr-asp6T1-9PrhbC-968XYj-968XR1-8Y2GHU-8awCiL-6T5Wc5-6T5VTY-5N5phd-JoycB" title="Models"><img src="https://live.staticflickr.com/214/479693617_34a96d6edd_w.jpg" width="400" height="318" alt="Models"></a><script async src="//embedr.flickr.com/assets/client-code.js" charset="utf-8"></script>

### Equilibrium and Tensegrity

If the twenty-four edges of eight hinge-bonded tetrahedrons thrust away in 12 directions from a common center, to the twelve corners the cuboctahedron, and twenty-four edges connect circumferentially in tension, preventing further expansion, then we get a model of equilibrium in Synergetics, the so-called "vector equilibrium" (VE).

The twisting-contracting motion of the Jitterbug Transformation turns the canonical cuboctahedron of volume 20, into the icosahedron of equal edge lengths and tetravolume ~18.512296.

When the compression components are "islanded" i.e. not touching one another, as tension members bind them together in a static arrangement, we get a tensegrity.

"Tensegrity" (coined by Fuller) provides a metaphoric model of a "system" (as in "systemic" and "systematic"), wherein time/size realized, energetic, tuned in, special case events occur in accordance with the exceptionless generalized principles running through and cohering them.

> The eternal is omniembracing and permeative; and the temporal is linear. This opens up a very high order of generalizations of generalizations. The truth could not be more omni-important, although it is often manifestly operative only as a linear identification of a special-case experience on a specialized subject. (1005.52)

> Truth is cosmically total: synergetic. Verities are generalized principles stated in semimetaphorical terms. Verities are differentiable. But love is omniembracing, omnicoherent, and omni-inclusive, with no exceptions. Love, like synergetics, is nondifferentiable, i. e., is integral. (1005.54)

Tensegrity sculptures, furniture, bridges etc. as a literal reality, were pioneered by Kenneth Snelson and others. Sam Lanahan took us back to the lattice in the form of flextegrity.

<a data-flickr-embed="true" href="https://www.flickr.com/photos/kirbyurner/16336137955/in/photolist-2kr3uUr-2j9GiPD-2i3t5Vo-qTyYEX-wsqaNG-aCHqW2-9Y6HBS-7r36Bg-7r36EM-7r37aH-7h9itS-5NFMD4-5xXjWP-7L5wXK" title="Tensegrity Favorite"><img src="https://live.staticflickr.com/7504/16336137955_1d6b9bb889.jpg" width="500" height="375" alt="Tensegrity Favorite"></a><script async src="//embedr.flickr.com/assets/client-code.js" charset="utf-8"></script>

### An Assessment 
##### By Kirby Urner

Calling *Synergetics* a Neoplatonic Metaphysics is not off the mark, but STEM is not Science only, and includes Mathematics, closer to Platonism than any empiricism.  There's a role for the *a priori*.  

Some critices keep insisting Synergetics is strictly empirical, which tends to mean statistical and data driven. True, Fuller favored the term *experiential* which included the intuitive (e.g. flashes of insight, gestalt shifts). Synergetics is open to the special case, but does not reduce to reductionism.  The anti-reductionist tone of Synergetics is what some scientists object to, but in this respect they're not speaking for all of Science, nor is *Synergetics* the only anti-reductionist philosophy out there.

> There are no solids. There are no things. There are only interfering and noninterfering patterns operative in pure principle, and principles are eternal. Principles never contradict principles. . . . The synergetic integral of the totality of principles is God, whose sum-total behavior in pure principle is beyond our comprehension and is utterly mysterious to us, because as humans — in pure principle — we do not and never will know all the principles. [Wikiquote](https://en.wikiquote.org/wiki/Buckminster_Fuller#Critical_Path_(1981))

How we summarize and present material, how we hybridize and alloy the topics together, makes a difference as well. How well is one able to visualize?  How about empathize, at some global gut level? 

Two teachers may have the same pile of Art & Science in front of them, to communicate to a next generation, but one of them is Bucky, and he gets his students to consider sphere packing and polyhedrons early, when they're still very young.  The other teacher shares the average more mediocre stuff we sometimes mistake for the real deal aka the "right stuff" and pushes polyhedrons off to the side somewhere, making sure they can't matter.

I would echo a natural science professors in saying "don't *just* read Synergetics" but I would not say "don't read *Synergetics*" i.e. I think Synergetics, a philosophy, a work in the humanities, continues to yield good ideas for STEM, worth exploring further and perhaps using architecturally, computationally, or in some other way.  

I don't think *Synergetics* + *Synergetics 2* should be treated as the one and only book one needs to read, as maybe too many treat their Bibles or whatever else they consider "holy".  Even if we admit Synergetics into the canon as a "sacred geometry" text, we are thereby only adding it to a still-growing library that already contains many lifetimes worth of reading and thinking.

Fuller felt he'd discovered a font, a fountain, a source, a cosmic zero, and wanted to pass it on to coming generations.  He considered it critical that more of us mine in the same vein.  He delivered his message with a sense of urgency.

> Quite clearly, our task is predominantly metaphysical, for it is how to get all of humanity to educate itself swiftly enough to generate spontaneous social behaviors that will avoid extinction. [Wikiquote](https://en.wikiquote.org/wiki/Buckminster_Fuller#The_Wellspring_of_Reality)

In retrospect, megatrends suggest he's attracting people who call themselves Designers more than he's attracting Scientists, but then we have the term "Design Science" to keep these two glued together, as well as [Designing Engineers](https://mitpress.mit.edu/books/designing-engineers).

[Martian Math on Wikieducator](https://wikieducator.org/Martian_Math)